src/lib/stream/sink/zenoh_sink.rs

```diff
@@ -201,8 +201,19 @@ impl ZenohSink {
             let zenoh_session = zenoh_session.clone();
             async move {
                 while let Some(data) = rx.recv().await {
+                    // Create a foxglove json compatible message
+                    // https://docs.foxglove.dev/docs/visualization/message-schemas/compressed-video
+                    let message = serde_json::json!({
+                        "timestamp": serde_json::json!({
+                            "sec": chrono::Utc::now().timestamp(),
+                            "nsec": chrono::Utc::now().timestamp_subsec_nanos(),
+                        }),
+                        "frame_id": "vehicle",
+                        "data": data,
+                        "format": "h264",
```

Comment:
Use the `encoding` to choose h264 or h265 here

Reply:
fixed